packages/protocol/migrations/23_elect_validators.ts

```diff
@@ -147,40 +183,38 @@ async function registerValidator(
     privateKeyToPublicKey(validatorPrivateKey)
   )
 
-  await sendTransactionWithPrivateKey(web3, registerDataEncryptionKeyTx, validatorPrivateKey, {
+  await sendTransaction(web3, registerDataEncryptionKeyTx, validatorPrivateKey, {
     to: accounts.address,
   })
 
-  // Authorize the attestation signer
-  const attestationKeyAddress = privateKeyToAddress(attestationKey)
-  console.info(`    - authorizeAttestationSigner ${valName}->${attestationKeyAddress}`)
-  const message = web3.utils.soliditySha3({
-    type: 'address',
-    value: privateKeyToAddress(validatorPrivateKey),
-  })
-  const signature = signMessage(message, attestationKey, attestationKeyAddress)
+  if (!isGanache) {
```

Comment:
Why not do this even when using ganache?

Reply:
I don't have enough keys for attestation signers.